prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -1146,29 +1146,33 @@ class Parallel : CoolTest() {
         options: CoolTestOptions
     ): Boolean {
         var passed = true
-        val elapsedMillis = measureTimeMillis {
+        var totalMillisAllSubmissions: Long = 0
+        val elapsedMillisTotal = measureTimeMillis {
             val threads = mutableListOf<Thread>()
             echo("Parallel Test: Starting $numThreads threads, each submitting $numRounds times")
             for (threadNum in 1..numThreads) {
                 val th = thread {
                     for (i in 1..numRounds) {
-                        val (responseCode, json) =
-                            HttpUtilities.postReportFile(
-                                environment,
-                                file,
-                                stracSender,
-                                options.key,
-                                ReportFunction.Options.SkipSend
-                            )
-                        if (responseCode != HttpURLConnection.HTTP_CREATED) {
-                            echo(json)
-                            passed = bad("$threadNum: ***Parallel Test FAILED***:  response code $responseCode")
-                        } else {
-                            val reportId = getReportIdFromResponse(json)
-                            if (reportId == null) {
-                                passed = bad("$threadNum: ***Parallel Test FAILED***:  No reportId.")
+                        val elapsedMillisOneSubmission = measureTimeMillis {
```

Comment:
Just wrapped this `./prime test --run parallel` test to print out more useful info on how fast it ran.